Listener/Notifier.php

```diff
@@ -94,7 +96,7 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         $exception = $event->getException();
 
         if ($exception instanceof HttpException) {
-            if (500 === $exception->getStatusCode() || (404 === $exception->getStatusCode() && true === $this->handle404) || (in_array($exception->getStatusCode(), $this->handleHTTPcodes))) {
+            if (!in_array($event->getRequest()->getClientIp(), $this->ignoredIPs) && (500 === $exception->getStatusCode() || (404 === $exception->getStatusCode() && true === $this->handle404) || (in_array($exception->getStatusCode(), $this->handleHTTPcodes)))) {
```

Comment:
this way all exceptions will be ignored not HTTP exceptions only.
do you see that it is better to ignore all exceptions not just the HTTP exceptions?